src/go/rpk/pkg/cli/cmd/wasm/common.go

```diff
@@ -105,3 +113,24 @@ func CreateRemoveMsg(name string) sarama.ProducerMessage {
 		Headers: headers,
 	}
 }
+
+func AddFlagType(command *cobra.Command, coprocType *string) {
+	command.Flags().StringVar(
+		coprocType,
+		"type",
+		"nodejs",
+		"Coproc type engine. Can be: nodejs(default), data-policy",
+	)
+}
+
+func CheckCoprocType(coprocType string) error {
+	coprocTypeSet := make(map[string]bool)
```

Comment:
I think a plain if statement here is cleaner, and simpler

Reply:
done